clifford/dcga.py

```diff
@@ -0,0 +1,123 @@
+"""
+The Cl(8,2) Double Conformal Geometric Algebra
+
+Easter, R.B., Hitzer, E. Double Conformal Geometric Algebra.
+Adv. Appl. Clifford Algebras 27, 2175–2199 (2017).
+https://doi.org/10.1007/s00006-017-0784-0
+"""
+
+from . import Layout
+
+# The metric of the algebra is two CGAs glued together
+layout = Layout([1]*4+[-1] + [1]*4+[-1])
+blades = layout.bases()
+
+locals().update(blades)
+
+
+# for shorter reprs
+layout.__name__ = 'layout'
+layout.__module__ = __name__
+
+
+# The two pseudo-scalars, infinities and origins
+IC1 = e12345
+IC2 = e678910
+einf1 = e4 + e5
+eo1 = 0.5*(e5 - e4)
+einf2 = e9 + e10
+eo2 = 0.5*(e10 - e9)
+
```

Comment:
Would it make sense to call the base blades `ea1, ea2, ...` and `eb1, eb2, ...`?

Reply:
Maybe, I have stuck with this way because it is similar to the way it is written in the paper:
https://link.springer.com/content/pdf/10.1007/s00006-017-0784-0.pdf
We could deviate from this for readability though

Reply:
Matching the paper seems like a good enough argument to me.